examples/post.v3.LR.amip.0101.cfg

```diff
@@ -0,0 +1,184 @@
+# To run:
+# Update unique_id in output, www below
+# > pip install . && zppy -c examples/post.v3.LR.amip.0101.cfg
+
+[default]
+input = /lcrc/group/e3sm2/ac.wlin/E3SMv3/AMIP/v3.LR.amip_0101
+output = /lcrc/group/e3sm/ac.forsyth2/zppy_pr719_output/unique_id_5/v3.LR.amip_0101
+www = /lcrc/group/e3sm/public_html/diagnostic_output/ac.forsyth2/zppy_pr719_output/unique_id_5
+campaign = "water_cycle"
+case = v3.LR.amip_0101
+environment_commands = "source /lcrc/soft/climate/e3sm-unified/load_latest_e3sm_unified_chrysalis.sh"
+partition = compute
+
+[ts]
+active = True
+walltime = "00:10:00"
+years = "2005:2015:5"
+
+  [[ atm_monthly_180x360_aave ]]
+  input_subdir = "archive/atm/hist"
+  input_files = "eam.h0"
+  frequency = "monthly"
+  mapping_file = /home/ac.zender/data/maps/map_ne30pg2_to_cmip6_180x360_aave.20200201.nc
+  # list for pcmdi diag, note: PHIS,hyam,hybm,hyai,hybi need to be included to process the 3D fields
+  vars = "ICEFRAC,LANDFRAC,OCNFRAC,PSL,FSNTC,FSNTOAC,SWCF,LWCF,FLUT,FSNT,FSNTOA,FLNT,FLNTC,FSNS,FLNS,FSNS,SHFLX,QFLX,LHFLX,TAUX,TAUY,PRECC,PRECL,PRECSC,PRECSL,TS,TREFHT,U10,QREFHT,TMQ,CLDTOT,CLDHGH,CLDMED,CLDLOW,FLDS,FSDS,TGCLDIWP,TGCLDCWP,TGCLDLWP,FLNSC,FLUTC,FSDSC,SOLIN,FSNSC,AODABS,AODVIS,AODDUST,AREL,TREFMNAV,TREFMXAV,PS,PHIS,U,V,T,Z3"
+
+[e3sm_to_cmip]
+active = True
+frequency = "monthly"
+ts_num_years=5
+walltime = "00:10:00"
+years = "2005:2015:5"
+
+  [[ atm_monthly_180x360_aave ]]
+  # list of variables to be interpolated to pressure level; see e3sm_to_cmip/vrt_remap_plev19.nc
+  cmip_plevdata = "inclusions/e3sm_to_cmip/vrt_remap_plev19.nc"
```

Comment:
This file doesn't exist in `zppy`'s `zppy/templates/inclusions/e3sm_to_cmip` path. Is it `inclusions` elsewhere?